include/seqan/basic/metaprogramming_enable_if.h

```diff
@@ -187,16 +187,16 @@ struct DisableIf<False, T>
 //    }
 
 template <bool b, typename T>
-struct EnableIf2;
+struct /*[[deprecated("Will be renamed to EnableIfC.")]]*/ EnableIf2;
```

Comment:
That's what the comments in this file said.. I thought to add the deprecation as a comment to be commented in as soon as EnableIfC is available.